litmus-portal/frontend/src/components/PredifinedWorkflows/data.ts

```diff
@@ -5,10 +5,10 @@ export default [
     urlToIcon:
       'https://hub.litmuschaos.io/api/icon/1.7.0/generic/node-cpu-hog.png',
     chaosWkfCRDLink:
-      'https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/node-cpu-hog/workflow.yaml',
+      'https://raw.githubusercontent.com/ishangupta-ds/chaos-workflows/master/Argo/argowf-native-node-cpu-hog.yaml',
 
     gitLink:
-      'https://github.com/litmuschaos/chaos-charts/blob/master/workflows/node-cpu-hog/workflow.yaml',
+      'https://github.com/ishangupta-ds/chaos-workflows/blob/master/Argo/argowf-native-node-cpu-hog.yaml',
```

Comment:
Please change the username `ishangupta-ds` to litmuschaos

Reply:
It won't work as these updated charts are not yet in the litmuschaos's chaos-charts repo. Me and @ksatchit are working in my repo and we will be sending a PR soon for adding workflows. These do not exist in litmuschaos org, changing the URL will break the frontend code. @rajdas98 cc: @arkajyotiMukherjee I will update it to litmuschaos when we have the changes in litmuschaos org probably in a couple of days.

Reply:
https://github.com/litmuschaos/litmus/pull/1967#discussion_r483437702

Reply:
Have removed these changes. @rajdas98